src/styles/main.scss

```diff
@@ -1,3 +1,75 @@
+@import './variables';
+
+* {
+  font-family: Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+h1 {
+  text-align: center;
+}
+
+.calendar {
+  margin: 0 auto;
+  padding: 20px;
+  width: $calendar-size;
+
+  display: flex;
+  gap: $distance-between;
+  flex-wrap: wrap;
+
+  $days: mon 0, tue calc($field-size + $distance-between), wed calc(($field-size * 2) + ($distance-between * 2)), thu calc(($field-size * 3) + ($distance-between * 3)), calc(($field-size * 4) + ($distance-between * 4)), sat calc(($field-size * 5) + ($distance-between * 5)), sun calc(($field-size * 6) + ($distance-between * 6));
```

Comment:
You missed to add day name 'fri' in $days list. It should be included to correctly calculate the margin for each day of week. Also you can make this with the help of the loop, and not to hardcode it like this.